trade-finance-manager-ui/templates/case/_macros/save-only-button.njk

```diff
@@ -0,0 +1,16 @@
+# This macro renders primary `Save` button only
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% macro render(params) %}
```

Comment:
the Govuk button is a component on its own. I'm not sure we need a separate component for that component?